string command
int32 age
float32 confidence
string add_object_type
string del_object_type
string color
string name
string location
string size_properties